pallets/offchain-worker/src/lib.rs

```diff
@@ -252,6 +257,19 @@ pub mod pallet {
 						let mut source_index = 0;
 						for source in &urls::DATA_SOURCE_LIST {
 							let task_index = urls::TOTAL_DATA_SOURCE_NUMBER * account_index + source_index;
+
+							// SBP M1 review: if I understand correctly, the logic is "distributing" the work
+							// over the various offchain workers, and a given account requesting a claim will
+							// always be processed by only 1 offchain worker.
```

Comment:
there are multiple off-chain workers for a single request. they submit the result separately. the final result will be chosen according to the amount.

Reply:
Concretely, this is the specific line that gets me confused:
`if task_index % ocw_length == ocw_account_index`
If every node (with ocw enabled) has a different account, then for a given task index, you will have the task processed by different ocw accounts if task indexes are (most of the time) > total number of off-chain workers, right ?
If the number of tasks processed is lower than the number of offchain worker, then only 1 worker will process every task. Maybe I got it completely wrong, please clarify how the load balancing is effectively working.

Reply:
If every node (with ocw enabled) has a different account, then for a given task index, you will have the task processed by different ocw accounts if task indexes are (most of the time) > total number of off-chain workers, right ?
Yes, and some of off-chain worker will take more than one task.

If the number of tasks processed is lower than the number of offchain worker, then only 1 worker will process every task. Maybe I got it completely wrong, please clarify how the load balancing is effectively working.
Yes, your understanding is completely correct. We just do the load balancing in this way.
I know it is not perfect, but I start the work from this simple algorithm. Will optimize it in the future.
Please let me know if you have better idea for load balancing.

Reply:
@suinuj All good for me, I just wanted to make sure that I got it right, and that you are aware of the various cases. I support going for something simple in a first iteration and improving in subsequent ones.